File: TenMinsJava/day20.java
// output fibonacci sequence (base 1) on each keypress
package TenMinsJava;

import java.util.Scanner;

public class day20 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        System.out.println("Enter to print fibonacci or q to quit");
        int a = 1;
        int b = 1;
        while (true) { 
            if (scan.nextLine().toLowerCase().equals("q")) {
                break;
            }
            else {
                System.out.print(a);
                int c = a + b;
                a = b;
                b = c;
            }
        }
    }
}

File: TenMinsPy/day20.py
# output fibonacci sequence (base 1) on each keypress

import keyboard

class day20:
    def __init__(self):
        inp = input("Enter to fibonacci and q to quit")
        a = 1
        b = 1
        while True:
            event = keyboard.read_event()
            if event.event_type == keyboard.KEY_DOWN:
                if event.name == 'enter':
                    print(a)
                    c = a + b
                    a = b
                    b = c
                elif event.name == 'q':
                    break;

day20()